config/fcos/v1_5_exp/schema.go

```diff
@@ -49,3 +49,20 @@ type GrubUser struct {
 	Name         string  `yaml:"name"`
 	PasswordHash *string `yaml:"password_hash"`
 }
+
+type Disk struct {
```

Comment:
Why are we adding Disk, Partiton to the fcos config? 

We have access to those types from the 
```
type Config struct {
	base.Config `yaml:",inline"`
	BootDevice  BootDevice `yaml:"boot_device"`
	Grub        Grub       `yaml:"grub"`
}
```

Reply:
I added it because when i tried to call Disk or Partition in validate.go or validate_test.go it would give me a "undeclared name" error on vscode. When i added the structs to schema.go it started working.
Is there a way to call the structs without adding them to the schema.go?

Reply:
https://github.com/coreos/butane/pull/436#discussion_r1131501088